3600/3/app/controllers/restaurant_controller.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+class RestaurantController < AppController
+  include CommentsHelper
+
+  get '/' do
+    @all_restaurants = Restaurant.all
+    erb :home
+  end
+
+  get '/restaurant/:name' do
+    session[:place] = params[:name]
```

Comment:
When you will did what said in a comment for 23rd line this line would unnecessary